app/org/sagebionetworks/bridge/services/ParticipantService.java

```diff
@@ -395,14 +397,19 @@ private IdentifierHolder saveParticipant(Study study, Set<Roles> callerRoles, St
         return new IdentifierHolder(account.getId());
     }
 
-    private Map<ParticipantOption, String> getAllOptionsButExternalId(Study study, StudyParticipant participant) {
+    private Map<ParticipantOption, String> getAllOptionsButExternalId(Study study, StudyParticipant participant,
+            boolean isNew) {
         Map<ParticipantOption, String> options = Maps.newHashMap();
         for (ParticipantOption option : ParticipantOption.values()) {
             options.put(option, option.fromParticipant(participant));
         }
         if (study.isExternalIdValidationEnabled()) {
             options.remove(EXTERNAL_IDENTIFIER);
         }
+        // You can't update time zone after it is set.
```

Comment:
Maybe changing the name from TIME_ZONE to something like ORIGINAL_EVENT_TIME_ZONE would be less confusing. Also, updating the comment to explain that we need to store the time zone in the participant, because we didn't store it in the event, and now we can't go back and change it. This means that someone can figure out what's going on by looking at the code instead of relying on tribal knowledge.